infrastructure/zk/src/run/forced-exit.ts

```diff
@@ -0,0 +1,129 @@
+            }
+        }
+        console.log('Forced exit sender account balance is not ready');
+        return;
+    }
+
+    console.log('Forced exit sender account balance is ready');
+}
+
+async function processRestRequest(nodeUrl: string, forcedExitAccount: string): Promise<RestAccountState> {
+    const response = await fetch(`${nodeUrl}/accounts/${forcedExitAccount}`);
+    const { result } = await response.json();
+
+    return result;
+}
+
+async function processJsonRpcRequest(nodeUrl: string, forcedExitAccount: string): Promise<RpcAccountState> {
+    const body = {
+        jsonrpc: '2.0',
+        method: 'account_info',
+        params: [forcedExitAccount],
+        id: 1
+    };
+    const response = await fetch(nodeUrl, {
+        method: 'POST',
+        body: JSON.stringify(body),
+        headers: {
+            Accept: 'application/json',
+            'Content-type': 'application/json'
+        }
+    });
+    const { result } = await response.json();
+
+    return result;
+}
+
+async function depositToForcedExitSenderAccount(
+    l1Address: string,
+    privateKey: string,
+    amount: string,
+    mnemonic: boolean
+) {
+    const forcedExitAccount = process.env.FORCED_EXIT_REQUESTS_SENDER_ACCOUNT_ADDRESS as string;
+    const provider = new ethers.providers.JsonRpcProvider(
+        process.env.FORCED_EXIT_REQUESTS_WEB3_URL ?? process.env.ETH_CLIENT_WEB3_URL
+    );
+    const wallet = mnemonic
+        ? ethers.Wallet.fromMnemonic(privateKey).connect(provider)
+        : new ethers.Wallet(privateKey).connect(provider);
+
+    if (l1Address.toLowerCase() !== wallet.address.toLowerCase()) {
+        console.log('L1 address does not match the provided private key or mnemonic');
+        return;
+    }
+
+    const mainZkSyncContract = new ethers.Contract(
+        process.env.CONTRACTS_CONTRACT_ADDR as string,
+        await utils.readZkSyncAbi(),
+        wallet
+    );
+
+    const depositTransaction = (await mainZkSyncContract.depositRBTC(forcedExitAccount, {
+        value: ethers.utils.parseEther(amount)
+    })) as ethers.ContractTransaction;
+
+    await depositTransaction.wait();
+
+    console.log('Deposit to the forced exit sender account has been successfully completed');
+}
+
+export const command = new Command('forced-exit').description('prepare forced exit sender account');
+
+command
+    .command('check')
+    .description('check forced exit sender account balance')
+    .arguments('<nodeUrl> <nodeType>')
+    .action(async (nodeUrl: string, nodeType: string) => {
+        if (nodeType !== 'REST' && nodeType !== 'JSONRPC') {
+            console.log('Node type must be either REST or JSONRPC');
+            return;
+        }
+        await checkForcedExitSenderAccountBalance(nodeUrl, nodeType);
+    });
+
+command
+    .command('prepare')
+    .description('deposit to forced exit sender account')
+    .arguments('<l1Address> <privateKey> [amount]')
+    .option('-m --mnemonic', 'Is mnemonic')
```

Comment:
instead of receiving a boolean to clarify if `privateKey` is a real private key or a mnemonic, I'd either use a private key only (to simplify the usage) or I'd accept either of them in two separate args `privateKey` or `mnemonic`

Reply:
updated